zendesk/deco
# Switching web framework from Play to Finatra, following John Bates' suggestion

:potable_water:

/cc @zendesk/sylvius

### Description

Switching from Play framework to Finatra.  Finatra code is copied from https://github.com/twitter/finatra/tree/master/examples/hello-world.

### How could this be manually tested?
* Pull code
* Start server by running `sbt run`
* Confirm controller works by visiting `http://localhost:8888/hi?name=foo`

### References
* JIRA: n/a

### Developer checklist
- [x] I have considered adding an Arturo for this change.
- [x] I have considered testing this branch against Classic.
- [x] I have added comments where applicable.

### Risks
* Low: nothing depends on this
